src/stories/pages/Login/Login.js

```diff
@@ -28,24 +29,18 @@ export const Login = ({ isDarkTheme }) => {
   const [error, setError] = useState('');
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const { setIsAuthenticated } = useAppContext();
 
   const getUser = (e) => {
     e.preventDefault();
 
-    const service = UserServices.getInstance();
-
-    const user = {
-      email,
-      password,
-    };
-    service.signIn(user).then((response) => {
+    signIn(email, password).then((response) => {
       if (response.errorCode) {
         // REDIRECT TO DASHBOARD
         // ERROR HANDLING
         setError(response.errorMessage);
       } else {
-        setError('');
-        window.alert(response.email);
+        setIsAuthenticated(true);
```

Comment:
Getting a TypeError on my end "'setIsAuthenticated' of 'Object(...)(...)' as it is null." Which causes login page to not render.

Reply:
My understanding is that 'setIsAuthenticated(true)' means that the user successfully logged in. If thats the case, do we need to still reroute to dashboard after it has set to true?

Also need to move line 39 'REDIRECT TO DASHBOARD' down to the else segment.

Reply:
This is outdated now, resolving